test/SIL/Serialization/reborrow.sil

```diff
@@ -0,0 +1,31 @@
+// First parse this and then emit a *.sib. Then read in the *.sib, then recreate
+
+// RUN: %empty-directory(%t)
+// RUN: %target-sil-opt %s -emit-sib -o %t/tmp.sib -module-name reborrow
+// RUN: %target-sil-opt %t/tmp.sib -module-name reborrow | %FileCheck %s
+
+sil_stage canonical
+
+class Klass {}
+
+// CHECK-LABEL: sil [serialized] [ossa] @reborrowTest : $@convention(thin) (@owned Klass) -> () {
+// CHECK: bb3([[ARG:%.*]] : @reborrow @guaranteed $Klass):
+// CHECK: } // end sil function 'reborrowTest'
+sil [serialized] [ossa] @reborrowTest : $@convention(thin) (@owned Klass) -> () {
```

Comment:
Did you want to add a similar test for printing/parsing in `test/SIL/Parser/borrow.sil`?

Reply:
Done.